### PR TITLE
NormalizeCommentsOnFuncGetNode

### LINKED ISSUE #488: [BUG]NormalizeCommentsOnFuncGetNode
**What is your environment(Kubernetes version, Fluid version, etc.)**

**Describe the bug**

**What you expect to happen:**

**How to reproduce it**

**Additional Information**

### PR DESCRIPTION
**Pull request by renke2020**
### Ⅱ. Does this pull request fix one issue?

fixes #488 

### Ⅴ. Special notes for reviews

### CODE REVIEW

**Comment by renke2020** [comment]
normalize comment on func GetNode(pkg/utils/kubeclient/node.go)

**Comment by renke2020** [comment]
normalize comment on func GetNode(pkg/utils/kubeclient/node.go)

File: pkg/utils/kubeclient/node.go [hunk 1]
```diff
@@ -24,7 +24,7 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
-// Get the latest node info
+//GetNode gets the latest node info
```
Comment (TrafalgarZZZ) [inline pkg/utils/kubeclient/node.go:27]: Please add a space between "//" and "GetNode"

**Review by TrafalgarZZZ — CHANGES_REQUESTED** [review]

File: pkg/utils/kubeclient/node.go (hunk 1 above)
Comment (renke2020) [reply pkg/utils/kubeclient/node.go]: Ok, I've completed the modification.

**Review by xieydd — APPROVED** [review]

**Pull request merged**